src/uiUpdater.ts

```diff
@@ -60,8 +60,8 @@ export function loopTasks(selectedProject: Project) {
             div.appendChild(taskDescription);
             div.appendChild(taskDeleteButton);
 
-            taskTitle.classList.add('taskTitle', `${task.title.replace(/\s/g, '').toLowerCase()}`);
-            taskDescription.classList.add( `${task.title.replace(/\s/g, '').toLowerCase()}Description`);
+            taskTitle.classList.add('taskTitle', `task${projectsSetup.projects[projectIndex].tasks.indexOf(task)}Title`);
```

Comment:
I missed this in the last pr, but are you wanting this style to only apply to that specific task/project? if you want a unique identifier for each project or task look into the `uuid` package

Reply:
`uuid` package seems useful but I don't think there is a need for such identifiers in this instance, and tailwind styles are applied elsewhere when the projects and tasks are made.